src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts

```
@@ -600,6 +614,34 @@ export class ResultsFiltersPreferences extends Component {
       customFilterType: type
     });
   }
+
+  private enlargeModalBox() {
+    const modalBoxContainer = $$(document.body).find('.coveo-modal-container');
+    if (modalBoxContainer) {
```

Multiple silent failures here